frigate/embeddings/onnx/lpr_embedding.py

```diff
@@ -0,0 +1,297 @@
+import logging
+import os
+import warnings
+
+import cv2
+import numpy as np
+
+from frigate.comms.inter_process import InterProcessRequestor
+from frigate.const import MODEL_CACHE_DIR
+from frigate.types import ModelStatusTypesEnum
+from frigate.util.downloader import ModelDownloader
+
+from .base_embedding import BaseEmbedding
+from .runner import ONNXModelRunner
+
+warnings.filterwarnings(
+    "ignore",
+    category=FutureWarning,
+    message="The class CLIPFeatureExtractor is deprecated",
+)
+
+logger = logging.getLogger(__name__)
+
+LPR_EMBEDDING_SIZE = 256
+
+
+class PlateDetectionEmbedding(BaseEmbedding):
```

Comment:
Might be better to call these `PaddleOCRDetectionEmbedding`, `PaddleOCRClassificationEmbedding`, and `PaddleOCRRecognitionEmbedding` since we decided to explicitly name Jina as well